docs/processes/documentation/documentation.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+Documentation is one of the most important parts of any software epic. It is imperative to remember, you will not be working on the capstone projects forever and that there will be people taking your place within Thoth Tech.
+
+Documentation is the offical information that is provided by the company for users and contributors. Documentation should provide clarity around troubleshooting and provide users and those involved with a basic idea of what it is the platform does and how it works.
+
+The Thoth Tech documentation should all be stored in the documentation repositry within the company GitHub. All documentation should be stored in a .md format and anything to do with research, meeting minutes and updates, should be stored within the [documentation repositry](https://github.com/thoth-tech/documentation).
+
+## Why is it important?
+
+Without documentation, there is no record of:
```

Comment:
Not entirely true, it is possible to find a record but it is difficult.

I would remove this section and the following list. Easier to link to find information in one spot such as the Docs repo.

Reply:
Section removed